server/common/default_config.yaml

```diff
@@ -0,0 +1,55 @@
+
+server:
+  verbose: true
+  debug: false
+  host: "127.0.0.1"
+  port : null
```

Comment:
shouldn't this by 5005 by default?

Reply:
I didn't change behavior.  The way it works is if port is null then it gets the starting port from the environment variable, and tries that.  If that one is taken, it tries the next one up until it finds a free port.  However if the port is specified (like 5005), then it only tries that port and fails if it is in use.

Reply:
oh, gotcha.  NM!